README.md

```diff
@@ -1,3 +1,4 @@
+
 # README
 
 To start the database in Docker:
```

Comment:
This first line could be a bit misleading for future recruits, because `dcoker-compose up` not just starts the database, but the production docker, which requires the volume created in the middle of the readme